### PR TITLE
Identifying info xml

### LINKED ISSUE #2263: Information about you JSON to eQIP XML
As an eApp developer I want to convert eApp **Information About You** form data in JSON format into the XML format used by the eQIP SOAP-based web service API so that I can submit the eApp form data via web service to the legacy eQIP database.

Acceptance Criteria
- [ ] data from eApp **Information About You** section JSON structure is converted to XML schema used by eQIP web service
- [ ] data can be transmitted via eQIP SOAP web service and saved in eQIP database

### PR DESCRIPTION
**Pull request by floresj**
Resolves #2263.

### CODE REVIEW

File: api/model/form/xml.go [hunk 1]
```diff
@@ -202,6 +204,14 @@ func daterange(data map[string]interface{}) template.HTML {
 	return xmlTemplateWithFuncs("date-range.xml", data, fmap)
 }
 
+func datecontrol(data map[string]interface{}) template.HTML {
+	fmap := template.FuncMap{
+		"date":          date,
+		"dateEstimated": dateEstimated,
+	}
+	return xmlTemplateWithFuncs("datecontrol.xml", data, fmap)
```
Comment (bmallred) [inline api/model/form/xml.go]: @floresj where is the `datecontrol.xml`?

File: api/model/form/templates/identification-othernames.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<OtherNamesUsed>
+    <Answer>{{ branch .props.HasOtherNames }}</Answer>
+    <Comment></Comment>
+    <HaveAdditionalEntryAnswer></HaveAdditionalEntryAnswer>
```
Comment (bmallred) [inline api/model/form/templates/identification-othernames.xml]: This is something we should probably clarify with @Jay-Neighbours-I72 if there should be a yes/no at the end of other names.

If the answer is to not have that then possibly hard code this to be "No"?

File: api/model/form/templates/identification-ssn.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{{ $ssn := .props.ssn }}
+<SSN NotApplicable="{{ $ssn.props.ssn.notApplicable }}">
```
Comment (bmallred) [inline api/model/form/templates/identification-ssn.xml]: This should probably be `$ssn.notApplicable` right?

File: api/model/form/templates/identification-othernames.xml (hunk 2 above)
Comment (floresj) [reply api/model/form/templates/identification-othernames.xml]: Good call. Sir @Jay-Neighbours-I72, thoughts on if we should add the Yes/No confirmation branch question at the end of the OtherNames accordion within the information about you section?
image

Comment (bmallred) [reply api/model/form/templates/identification-othernames.xml]: May push this PR through but try to keep track of this

**Pull request merged**